rust_icu_uloc/src/lib.rs

```diff
@@ -361,6 +362,23 @@ impl ULoc {
         )
     }
 
+    /// Returns the collection of available locales via 'uloc_countAvailable'/'uloc_getAvailable'
```

Comment:
Add a formulaic /// Implements `...` here, for coverage.

Also if you are able can you run `make cov` to update the coverage data?
`make cov` is not hermetic and requires `exuberant-ctags`. If you don't have that package or can't install it, feel free not do it.

Reply:
Will add the 'Implements' comment and see if I can run the 'make cov'

Reply:
I installed exuberant-ctags but the output in my dev environment (Mac OS) doesn't look correct, so not including that.